public_dropin_gpu_environments/triton_server/model_config/README.md

```diff
@@ -0,0 +1,7 @@
+This folder contains the Triton model configuration parser generated from the protocol buffer scheme (r24.02)
```

Comment:
Is it really 24.02? I thought we had to revert back to 24.01 due to Onnx issue?

Reply:
yep. I took scheme from the 24.02, but it works with 24.01 w/o issues